go/state/state_db.go

```diff
@@ -14,12 +14,13 @@ package state
 import (
 	"errors"
 	"fmt"
-	"github.com/Fantom-foundation/Carmen/go/common/witness"
 	"maps"
 	"math/big"
 	"sync"
 	"unsafe"
 
+	"github.com/Fantom-foundation/Carmen/go/common/witness"
+
```

Comment:
this change is unrelated, please revert.

I understand that it was done by IDE, and for this reason we accept such changes when there are real functional changes in the file. But in this case nothing has changed